CMakeLists.txt

```diff
@@ -21,28 +21,55 @@
 # along with vsUTCS.  If not, see <https://www.gnu.org/licenses/>.
 
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.5 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.16 FATAL_ERROR)
 
 SET(vsUTCS_VERSION_MAJOR "0")
-SET(vsUTCS_VERSION_MINOR "9")
+SET(vsUTCS_VERSION_MINOR "10")
```

Comment:
did I miss this on master?

Reply:
I'm not sure. All I know is that I found it set to 0.9.0 still. 😄